python/ray/util/annotations.py

```diff
@@ -107,22 +113,52 @@ def Deprecated(*args, **kwargs):
     if len(args) == 1 and len(kwargs) == 0 and callable(args[0]):
         return Deprecated()(args[0])
 
-    message = (
-        "**DEPRECATED:** This API is deprecated and may be removed in a future "
-        "Ray release."
+    doc_message = (
+        "\n    DEPRECATED: This API is deprecated and may be removed "
+        "in future Ray releases."
+    )
+    warning_message = (
+        "This API is deprecated and may be removed in future Ray releases. "
+        "You could suppress this warning by setting env variable "
+        "RAY_DISABLE_DEPRECATION_WARNING=0."
     )
 
+    warning = kwargs.pop("warning", False)
```

Comment:
is there a reason why we don't turn on warning by default?

Reply:
yeah, let's change incrementally.